recipes/stemflow/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "stemflow" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 194af6f1247875eb7e0eadbec327f29e29d569484337cc45e003b7a565315814
+
+build:
+  number: 0
+  skip: true  # [win, osx]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```

Comment:
`host` typically only requires `pip`, `python`, `setuptools` and `setuptools_scm`.

Reply:
Also, I don't see `setuptools_scm` used anywhere in this project. You likely only need `python` and `pip`.

Reply:
Changed host to 
```
  host:
    - pip
    - python >=3.8
```